process/charter.html

```diff
@@ -81,189 +77,188 @@ <h4>Tools and templates</h4>
-  to start dedicating resources.</p> -->
+<dl>
+  <dt>Check readiness</dt>
+  <dd>For a Working Group charter, review <a href="https://www.w3.org/Guide/standards-track/">W3C Recommendation Track Readiness Best Practices</a> and <a href="/2002/05/rec-tips">Tips for Getting to Recommendation Faster</a>.</dd>
+  <dt>Draft charter</dt>
+  <dd>Use the <a href="https://w3c.github.io/charter-drafts/charter-template.html">charter template</a> to create a draft charter, ideally on GitHub. This is where substantive discussion of the charter should take place, including issues and pull requests. <strong>Note:</strong> There are ongoing discussions about making it easier for people to find draft charters, e.g., by using a single GitHub repo for all draft charters.</dd>
+  <dt>Inform Strategy Team</dt>
+  <dd>When ready (e.g., after sufficient discussion among stakeholders has taken place), inform the Strategy Team by
+<a href="https://github.com/w3c/strategy/issues/new?assignees=&labels=Evaluation%3A+untriaged&template=04-Chartering.md&title=">creating a new charter issue</a> in their <a href="#pipeline">pipeline</a>. The Strategy Team will use this issue to provide updates on the charter's progress through the process. Discussions of the charter's content should continue to take place in the charter's own repo.</dd>
+</dl>
 
-<h2 id="advance">1. New Groups: Advance Notice</h2>
+<h3 id="strategy-team">Strategy Team Role</h3>
 
-<!-- Resolved at W3M 11 July 2007 -->
+<p>When a new charter issue has been raised, the Strategy Team Lead assigns a staff member to <dfn id="charter-shepherd">shepherd</dfn> the charter through the remainder of the early process. The Strategy Team typically discusses proposed charters at its regular meeting or on its mailing list. <strong>Note:</strong> If the Strategy Team Lead cannot identify a staff member, there may be delays in advancing the charter through subsequent reviews.</p>
 
-<p><a
-href="https://www.w3.org/Consortium/Process/#WGCharterDevelopment">W3C Process</a> requires Team notify the Advisory Committee when a
-charter for a <strong>new</strong> Working Group or Interest Group is
-in development. The exact timing of the advance notice may vary from
-charter to charter. In practice, if the advance notice would precede
-the formal call for review by only a short delay, we skip the advance
-notice.</p>
+<p>The <a href="#charter-shepherd">charter shepherd</a> (on behalf of the Strategy Team) roles include:</p>
 
-<p>The Team should consult <a href="adv-notice.html">How to
-Send Advance Notice of Work to the Advisory Committee</a> for more
-information and a template announcement.</p>
+<dl>
+  <dt>Evaluate readiness</dt>
+  <dd>Let the proponents know if the charter is not well-formed (per the template), if it is perceived to be harmful to the Web, or if it is not a priority at the current time.</dd>
+  <dt>Send advance notice to AC</dt>
```

Comment:
I'd suggest to change "Send advance notice to AC" to "Work with W3C Communications Team to send advance notice to AC". We normally don't expect team contacts to directly send advance notices, and sending emails to <ac-members> (different from <ac-forum>) seems restricted to a sub-group of the Team.

Reply:
```suggestion
  <dt>Work with W3C Communications Team to send <a href="adv-notice.html">advance notice to AC</a></dt>
```

Reply:
Hi @xueyuanjia, 
I agree with the point that the team contact does not do the advance notice. But I would like to avoid mentioning details in the short label, and I would like to keep the short label very short. I propose instead: "Request advance notice to AC."